homepage/scripts/scripts.js

```diff
@@ -216,8 +216,9 @@ function loadStyles() {
 
 (async function loadPage() {
   loadStyles();
-  const { loadArea, setConfig } = await import(`${miloLibs}/utils/utils.js`);
+  const { loadArea, setConfig, loadLana } = await import(`${miloLibs}/utils/utils.js`);
   setConfig({ ...CONFIG, miloLibs });
+  loadLana({ clientId: 'homepage' });
```

Comment:
We need to check if clienId has setup.

Reply:
Here is PR for adding homepage as a clientId on LANA side,
https://git.corp.adobe.com/wcms/lana/pull/14